mods/ra/chrome/settings.yaml

```diff
@@ -203,6 +203,13 @@ Background@SETTINGS_PANEL:
 					Height: 20
 					Font: Regular
 					Text: Always Show Status Bars
+				Checkbox@DISPLAY_TARGET_LINES_CHECKBOX:
+					X: 310
+					Y: 260
```

Comment:
Please make this 265 so that the spacing matches the rest of the UI. The cnc one is fine, it uses different spacing.